### PR TITLE
Add Credits Page

### PR DESCRIPTION
**Pull request by KevinCawley**
Take the Credits and Publication policies page from the docs and add it here. 

**Description**

**Motivation and context**

**How has this been tested?**
- [ ] Testing pipeline.
- [x] Other. 

**Examples**

**Type of change**

- [ ] Bug fix. 
- [x] New feature. 
- [ ] Breaking change. 
- [ ] None of the above. 

**Checklist**

- [ ] My change requires a change to the documentation.
    - [ ] I have updated the documentation accordingly.
    - [ ] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html).
- [ ] I have assigned and requested two reviewers for this pull request.

### CODE REVIEW

**Comment by KevinCawley** [comment]
Right now the software isn't in bibtex quotes. This is because, right now, I am unsure if I will be including the contents or if it will be a link to a file with the information. I can make it in bibtex if we want to keep it as-is right now. However, that will be a commit.

**Comment by KevinCawley** [comment]
@jaladh-singhal If you want, I can open a new PR that will only have 1 commit by doing `git fetch upstream` and `git checkout upstream/master`, and just modifying it again. I had made this branch by doing a checkout from the master!

File: content/credits.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+title: "Credits"
+date: 2021-05-28T10:09:58-05:00
+draft: false
+layout: page
+---
+
+
+# Credits & Publication Policies
```
Comment (jaladh-singhal) [inline content/credits.md:9]: Remove this from here and put it in `title` instead (at L2)

File: content/credits.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+---
+title: "Credits"
```
Comment (jaladh-singhal) [inline content/credits.md:2]: Change it to Credits & Publication Policies

**Review by jaladh-singhal — CHANGES_REQUESTED** [review]

**Review by jaladh-singhal — APPROVED** [review]

**Pull request merged**